docs(models): naming convention info (#112)
nextauthjs/docs: docs/adapters/prisma.md

```
npx prisma migrate dev
```

## Naming Conventions

> If mixed snake case and camel case column names is an issue for you and/or your underlying database system, we recommend using Prisma's `@Map()` feature to change the field names. This won't affect NextAuth, but will allow you to customize the column names to whichever naming convention you wish.

Reviewer: ```suggestion
If mixed snake_case and camelCase column names are an issue for you and/or your underlying database system, we recommend using Prisma's `@Map()` feature to change the field names. This won't affect NextAuth.js but will allow you to customize the column names to whichever naming convention you wish.
```
